packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -514,7 +514,11 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
               calcTargetRatio(),
             );
 
-            if (state.remainingProceedsGoal !== null) {
+            if (
+              state.remainingProceedsGoal !== null ||
+              // XXX use this as an indication that the auction is active
```

Comment:
if we end up revving CI, please elaborate the comment. (auction has an active state but the book doesn't have access to it)